x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/job_map/components/legend.tsx

```diff
@@ -43,10 +43,10 @@ export const JobMapLegend: FC<{ hasMissingJobNode: boolean }> = ({ hasMissingJob
   const euiColorFullShade = euiTheme.colors.fullShade;
   const euiColorGhost = euiTheme.colors.ghost;
   const euiColorWarning = euiTheme.colors.warning;
-  const euiBorderThin = euiTheme.border.thin;
+  const euiBorderThin = euiTheme.border.width.thin;
```

Comment:
Question: `euiBorderThin` does map to `euiTheme.border.thin` - Is this change expected/wanted because the usage cahnged for this being required to only be `1px` instead of `1px solid {euiTheme.border.color}`?

Reply:
Well spotted, you're right, this change was not intended, I reverted it in [dc5b902](https://github.com/elastic/kibana/pull/203518/commits/dc5b902e5874c0797aa38ab9d60976298c249302).